src/EFCore.PG/Storage/Internal/Mapping/NpgsqlRangeTypeMapping.cs

```diff
@@ -9,21 +9,41 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.Mapping
 {
     public class NpgsqlRangeTypeMapping : NpgsqlTypeMapping
     {
+        [NotNull] static readonly NpgsqlSqlGenerationHelper SqlGenerationHelper =
+            new NpgsqlSqlGenerationHelper(new RelationalSqlGenerationHelperDependencies());
+
+        [CanBeNull] readonly string _storeTypeSchema;
```

Comment:
Let's put attributes on the line before a field declaration like we do it for properties.

Reply:
So, I don't feel too strongly one way or another, but I think the inline version for fields is used throughout the project. Mind if I leave it as is for now?

Reply:
If it's projectwise, then okay.